src/Server/Remote/NativeSsh.php

```diff
@@ -230,12 +206,55 @@ protected function getConnectionHash()
     protected function getMultiplexingSshOptions()
     {
         return [
-            '-o ControlMaster=auto',
-            '-o ControlPersist=5',
-            '-o ControlPath=\'' . $this->getConnectionHash() . '\'',
+            '-o' => [
+                'ControlMaster=auto',
+                'ControlPersist=5',
+                'ControlPath=' . $this->getConnectionHash(),
+            ],
         ];
     }
 
+    /**
+     * Returns final ssh connection command with configs, username and host.
+     *
+     * @return array
+     */
+    private function getSshConnectionOptions()
```

Comment:
I think this these options should be reusable. Maybe make this method public?
For example, I use rsync in my docker.php and it also needs these options. How should I pass them into it?